pkg/printers/internalversion/printers.go

```diff
@@ -72,10 +72,12 @@ const (
 
 	// labelNodeRolePrefix is a label prefix for node roles
 	// It's copied over to here until it's merged in core: https://github.com/kubernetes/kubernetes/pull/39112
-	labelNodeRolePrefix = "node-role.kubernetes.io/"
+	labelNodeRolePrefix           = "role.node.kubernetes.io/"
```

Comment:
Where this is coming from? I can't seem to find that label anywhere in the code?

Reply:
From the description of this PR:
> I suggest that `node-role.kubernetes.io` can be changed to `role.node.kubernetes.io`, `kubernetes.io/role` can be changed to `kubelet.kubernetes.io/role`

Also in the description:
> In v1.17.3, the label prefix `node-role.kubernetes.io` is not allowed, suggesting `node.kubernetes.io` instead.

That one definitely needs to be prefixed as `node.kubernetes.io` is used for other purposes (e.g. `node.kubernetes.io/instance-type`) - as all prefixes are allowed by https://github.com/kubernetes/enhancements/blob/master/keps/sig-auth/0000-20170814-bounding-self-labeling-kubelets.md#proposal, `role.node.kubernetes.io` seems a great choice)